src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,10 @@
+const errorHandler = ((err, req, res, next) => {
+    console.log('Error', err);
+    res.send({
+        error: 'not found',
+        message: 'error',
+        status: 500,
+        timestamp: new Date()
```

Comment:
move all these to a variable and pass them into response

Reply:
Done